.babelrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
maybe lets hold off on styled-jsx, it might be better to do our styles in regular CSS since we have to work also with legacy code